x/ecocredit/simulation/genesis.go

```diff
@@ -141,26 +152,25 @@ func genBalances(r *rand.Rand,
 	return result
 }
 
-func getBatchbyClassID(batches []*ecocredit.BatchInfo, classID string) *ecocredit.BatchInfo {
+func getBatchbyProjectID(batches []*ecocredit.BatchInfo, projectID string) *ecocredit.BatchInfo {
```

Comment:
```suggestion
func findBatchByProjectID(batches []*ecocredit.BatchInfo, projectID string) *ecocredit.BatchInfo {
```

Reply:
I think we should stick to the consistent use of `get`. We are returning `BatchInfo` here.